### PR TITLE
Gomez script

### PR DESCRIPTION
**Pull request by cag-martinez**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h1>Music +</h1>
             <div class="row-6"></div>
             <input id="submit" type="submit"></input>
             </div> 
-        </div>  
+       
             
         </div>
```
Comment (cglennon924) [inline index.html:27]: line it up with container div

File: index.html [hunk 2]
```diff
@@ -45,7 +45,7 @@ <h1>Music +</h1>
     <script src= "api.js" type="text/javascript"> </script>
     <!-- Here i had to add source for jquery-->
     <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
-
+</div>
```
Comment (cglennon924) [inline index.html:48]: get rid of this div

**Review by cglennon924 — CHANGES_REQUESTED** [review]
C:\Users\cglen\Documents\Project-1\Project-1\index.html

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    
    <title>Music +</title>
    <link rel="stylesheet" type="text/css" href="Mainpage.css">
    <link href="https://fonts.googleapis.com/css2?family=Squada+One&display=swap" rel="stylesheet">
</head>

<body>
    <h1>Music +</h1>
        
    <div class="container">
        <form action="./index2.html">
        <input type="text" id="search-musician" placeholder="John Lennon"></input>
        
            
        <button id="submit" type="submit">Search</button>
        <button id="clear" type="clear">Clear</button>
        </form>
                  
    <div id="output"></div>
    <p class="description">
        Welcome to Music +! Type in the searchbar above to find out more information about your favorite musician! You will be surprised at what you learn and also can sing along while you read the lyrics.
    </p>
     
    </div>
    
    <script src= "api.js" type="text/javascript"> </script>
    
    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
    
    

</body>

</html>

this is what mine looks like

**Review by cag-martinez — COMMENTED** [review]
fixed based on the comment

**Review by cglennon924 — APPROVED** [review]

**Pull request merged**